docs/best-practices/serverless/azure.md

```diff
@@ -142,7 +142,7 @@ https://github.com/DevCycleHQ/azure-functions-example
 With the [DevCycle NodeJs SDK ](/sdk/server-side-sdks/node) we use the `enableCloudBucketing` and `enableEdgeDB` options to enable EdgeDB usage. ([SDK initialization option details: ](/sdk/server-side-sdks/node/node-gettingstarted#initialization-options))
 
 ```jsx
-const dvcClient = DVC.initialize(serverKey, {
+const devcycleClient = DVC.initialize(serverKey, {
```

Comment:
We should probably create another epic to go an clean up all those tertiary examples once day like https://github.com/DevCycleHQ/azure-functions-example/ which introduce these DVC references